beacon-chain/forkchoice/doubly-linked-tree/proposer_boost_test.go

```diff
@@ -126,8 +126,8 @@ func TestForkChoice_BoostProposerRoot_PreventsExAnteAttack(t *testing.T) {
 				ctx,
 				slot,
 				newRoot,
-				headRoot,
-				params.BeaconConfig().ZeroHash,
+				indexToHash(2),
```

Comment:
if we want block to be built on 2 then the parent root should be `indexToHash(2),`